### DISCUSSION TITLE
Repositório sobre React.js

### DISCUSSION THREAD

**Discussion by pietrobelluno** [frontendbr/forum#1023] [Perguntas e Respostas]
Fala galera , recentemente comecei a mexer com React.js e não achei nenhum awesome React.js ( só de React Native). Então quero criar essa issue para compartilharmos tutoriais, artigos, cursos, video-aulas, livros sobre React.js para que posteriormente eu junte as informações e crie um repositório com o máximo de conteudo sobre React. Conto com a ajuda de vocês !

Começando , segue o link do tutorial que usei para começar a aprender React:

https://www.youtube.com/watch?v=MhkGQAoc7bc&list=PLoYCgNOIyGABj2GQSlDRjgvXtqfDxKm5b

**Comment by ghost** [comment]
@pietrobp o mais famoso de awesome de React é esse:
https://github.com/enaqx/awesome-react

há muita contribuição e algumas discussões interessante.

Bacana que você criou algo a mais pra agregar. =)

**Comment by pietrobelluno** [comment]
@edmolima psé eu vi ele depois, mas pra falar bem a verdade ele é muito confuso, eu queria fazer um repositório mais simples e de entendimento mais fácil

**Comment by lu-zen** [comment]
@pietrobp Como vai sua caminhada com o React?

**Comment by Kecbm** [comment]
React é uma ferramenta muito poderosa, é incrível desenvolver o frontend com ele 🤩
